### PR TITLE
setSelection feature

### PR DESCRIPTION
**Pull request by zefrog**
Following the PR #1382 and your remarks about a new `setSelection` method, this is a proposal for such a method, keeping backward compatibility.

Let me know if this is alright.

Regards.

### CODE REVIEW

File: lib/network/modules/SelectionHandler.js [hunk 1]
```diff
@@ -601,6 +601,44 @@ class SelectionHandler {
...
 
+  /**
+   * Updates the current selection
+   * @param {{nodes: Array.<String>, edges: Array.<String>}} Selection
+   * @param {Object} options                                 Options
+   */
+  setSelection(selection, options = {}) {
+    let i, id;
+
+    if (!selection || !selection.nodes || !selection.edges)
```
Comment (josdejong) [inline lib/network/modules/SelectionHandler.js]: For convenience, we could allow only providing one of the two: `nodes` or `edges`, i.e. allow `Network.setSelection({nodes: [1,2,3]})`.

**Comment by josdejong** [comment]
Thanks @zefrog , nice work. I added one inline comment but besides that it looks very neat.

**Comment by zefrog** [comment]
Ok I considered your comment and rewrite a bit the function. I also removed the vars in selectEdges and selectNodes methods which were no longer used.

**Comment by josdejong** [comment]
Thanks @zefrog !

**Pull request merged**